### PR TITLE
Badge replacement

### PR DESCRIPTION
**Pull request by adcroft**
Some minor tidy ups to the Makefile so I can play with more fine grained badges. No functional changes to usual operations.

### CODE REVIEW

File: .testing/Makefile [hunk 1]
```diff
@@ -321,7 +328,7 @@ results/%/ocean.stats.restart: ../build/symmetric/MOM6
 clean: clean.stats
 	@# Assert that we are in .testing for recursive delete
 	@[ $$(basename $$(pwd)) = .testing ]
-	rm -rf ../build
+	rm -rf ../build ../deps/fms
```
Comment (marshallward) [inline .testing/Makefile]: In nearly every case, it's not necessary to clear the FMS build, so I've been reluctant to include this.  I'd rather find a way to get the Makefile to be smart enough to detect if FMS source has changed.  And rebuilding FMS can be expensive.

A big bottleneck here is that this Makefile doesn't see the changes detected by the actual FMS makefile from mkmf.  I currently don't know how to solve that.

File: .testing/Makefile [hunk 2]
```diff
@@ -88,6 +88,13 @@ SOURCE = $(wildcard $(BASE)/src/*/*.F90 $(BASE)/src/*/*/*.F90 $(BASE)/config_src
 all: $(foreach b,$(BUILDS),$(BUILD)/$(b)/MOM6)
```
Comment (marshallward) [inline .testing/Makefile]: Might as well edit this to use the rules defined below.

File: .testing/Makefile [hunk 3]
```diff
@@ -88,6 +88,13 @@ SOURCE = $(wildcard $(BASE)/src/*/*.F90 $(BASE)/src/*/*/*.F90 $(BASE)/config_src
...
 build.regressions: $(foreach b,symmetric target,$(BUILD)/$(b)/MOM6)
 
+# Shortcuts to build executables
+build-symmetric: $(BUILD)/symmetric/MOM6
+build-asymmetric: $(BUILD)/asymmetric/MOM6
+build-openmp: $(BUILD)/openmp/MOM6
+build-repro: $(BUILD)/repro/MOM6
+build-fms: $(FMS)/lib/libfms.a
+
```
Comment (marshallward) [inline .testing/Makefile]: build.symmetric, build.asymmetric, etc to match the other rules?

File: .testing/Makefile (hunk 2 above)
Comment (adcroft) [reply .testing/Makefile]: I wasn't meaning to change the live rules, just add some shortcuts/

File: .testing/Makefile (hunk 1 above)
Comment (adcroft) [reply .testing/Makefile]: That's fine. It's just for interactive we sometimes need to do a real clean. In the old days we used "clean" "Clean" and "CLEAN" for degrees of cleaning.

File: .testing/Makefile (hunk 3 above)
Comment (adcroft) [reply .testing/Makefile]: Once you get rid of the paths wouldn't the rule b build/symmetric ?

Comment (marshallward) [reply .testing/Makefile]: build/symmetric/MOM6 but yeah basically

**Pull request closed without merging**